rmgpy/molecule/symmetry.py

```diff
@@ -113,7 +113,7 @@ def calculateAtomSymmetryNumber(molecule, atom):
             # Two single bonds
             if count == [2]:
                 symmetryNumber *= 2
-    
+
```

Comment:
Definitely. I just make this to try to re-trigger RMG-tests, since it wasn't functioning properly.